hack/tools/go.mod

```diff
@@ -15,3 +15,236 @@ require (
 	sigs.k8s.io/kustomize/kustomize/v4 v4.4.0
 	sigs.k8s.io/testing_frameworks v0.1.2
 )
+
+require (
+	4d63.com/gochecknoglobals v0.1.0 // indirect
```

Comment:
is this intake expected? all coming from golangci-lint?

Reply:
Most of them are because we updated the golang version to 1.18 which basically brings in the Module graph pruning changes (from go@1.17)